File: module-android-api/src/main/java/com/tainzhi/sample/api/widget/HorizontalSpaceItemDecoration.java
package com.tainzhi.sample.api.widget;

import android.graphics.Rect;
import android.view.View;

import androidx.annotation.NonNull;
import androidx.recyclerview.widget.RecyclerView;

/**
 * Created by muqing on 2019-08-15.
 * Email: qfq61@qq.com
 */
public class HorizontalSpaceItemDecoration extends RecyclerView.ItemDecoration {
	private int space;
	
	public HorizontalSpaceItemDecoration(int space) { this.space = space; }
	
	;
	
	@Override
	public void getItemOffsets(@NonNull Rect outRect, @NonNull View view, @NonNull RecyclerView parent, @NonNull RecyclerView.State state) {
		if (parent.getChildLayoutPosition(view) != 0) {
			outRect.left = space;
		}
	}
}

File: module-android-api/src/main/java/com/tainzhi/sample/api/widget/HorizontalSpaceItemDecoration.kt
package com.tainzhi.sample.api.widget

import android.graphics.Rect
import android.view.View
import androidx.recyclerview.widget.RecyclerView
import androidx.recyclerview.widget.RecyclerView.ItemDecoration

/**
 * Created by muqing on 2019-08-15.
 * Email: qfq61@qq.com
 */
class HorizontalSpaceItemDecoration(private val space: Int) : ItemDecoration() {
    override fun getItemOffsets(
        outRect: Rect,
        view: View,
        parent: RecyclerView,
        state: RecyclerView.State
    ) {
        if (parent.getChildLayoutPosition(view) != 0) {
            outRect.left = space
        }
    }

}